portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java

```diff
@@ -87,9 +87,13 @@ protected void processFilter(
 		httpServletResponse.setHeader(
 			HttpHeaders.CACHE_CONTROL, "private, no-cache");
 
+		String eTagValue = _eTagValues.get(eTagKey);
+
 		if ((ifNoneMatch != null) &&
 			ifNoneMatch.equals(_eTagValues.get(eTagKey))) {
 
```

Comment:
I'd be clearer if we use `eTagValue` instead of `_eTagValues.get(eTagKey)` in this comparison

Reply:
nvm you change this in the next commit